### DISCUSSION TITLE
Possible Staging Environment?

### DISCUSSION THREAD

**Discussion by michaelmerrill** [vercel/vercel#5419] [Ideas] [answered]
Any thoughts on adding a dedicated staging environment to Vercel outside of production, preview, and development? Looking through the issues and articles online, I believe the community would really use it.

**Comment by styfle** [comment, marked as answer]
Hi @michaelmerrill 

We just announced this feature: https://vercel.com/changelog/environments-variables-per-git-branch

Also checkout this guide: https://vercel.com/support/articles/set-up-a-staging-environment-on-vercel
